### ISSUE TITLE
Warning about missing automake or aclocal-1.14

### ISSUE THREAD

**Issue by andy5995** [andy5995/rmw#21]
Jammyjamjamman reported this problem today after attempting to build the code from master on Ubuntu

>  WARNING: 'automake-1.14' is missing on your system.
> You should only need it if you modified 'Makefile.am' or
> 'configure.ac' or m4 files included by 'configure.ac'.

> The 'automake' program is part of the GNU Automake package:
> <http://www.gnu.org/software/automake>
> It also requires GNU Autoconf, GNU m4 and Perl in order to run:
> <http://www.gnu.org/software/autoconf>
> <http://www.gnu.org/software/m4/>
> <http://www.perl.org/>
> Makefile:330: recipe for target 'Makefile.in' failed
> make: *** [Makefile.in] Error 

Thanks, Jammy.

**Comment by andy5995** [comment]
When 'Makefile' is present, running 'make' doesn't trigger the attempt to run 'automake'.

But now there are lines such as these in the Makefile, with paths that aren't present for users who are building:
'ACLOCAL = ${SHELL} /home/andy/src/rmw/missing aclocal-1.14'

That doesn't seem to affect the build though. I switched to root, cloned the repo, ran './configure' and 'make'. 'automake' wasn't triggered unless I ran 'make' before running configure.

**Comment by andy5995** [comment]
After I changed this:
`rmw_CFLAGS =  -std=gnu99 -Wall -I./functions  -DSYSCONFDIR=\"${sysconfdir}\"`

to this:
`AM_CFLAGS= -std=gnu99 -Wall -I./functions  -DSYSCONFDIR=\"${sysconfdir}\"`

the warning went away.

**Comment by andy5995** [comment]
I think that problem was related to this:

https://github.com/tstack/lnav/issues/54

And my solution (until it stops working) is this:

I was got frustrated with this problem when I started using travis. I think I figured it out though.

After running the automake tools, the build still failed. I thought I had to edit the first line in configure, missing, depcomp, etc from #!/bin/sh to #!/bin/bash. But from what I read, that is really frowned upon.

So now, it seems that if after I run the automake tools, when I run ./configure && make && make clean && make distclean and then push my changes, travis will build my stuff.

So basically, you don't want to do a distclean, and then run your automake tools without rebuilding and cleaning again.

**Comment by andy5995** [comment]
Nope, that wasn't the problem. It worked a few times consecutively, and then the problem popped up again.

**Comment by andy5995** [comment]
This is actually happening on travis. I don't know if it would happen every time for a regular Ubuntu User. And the message is _WARNING: 'aclocal-1.14' is missing on your system._

**Comment by andy5995** [comment]
This problem was just reported by an Ubuntu 16.04 user.

**Comment by parthsuresh** [comment]
This solved the problem on my Ubuntu 16.04 system:
`sudo apt-get install autotools-dev
 sudo apt-get install automake`

Then run these commands in the source folder:
`aclocal
automake
./configure
make
`

**Comment by andy5995** [comment]
This appears to be fixed. If it has any unintended side effects for any users, I'll reopen the issue.
